frontend_lib/src/action.async.js

```diff
@@ -68,3 +68,35 @@ export const putContentRestoreDelete = (apiUrl, workspaceId, contentId) => {
     method: 'PUT'
   })
 }
+
+export const getUsernameAvailability = async (apiUrl, username) => {
+  const response = await fetch(`${apiUrl}/system/username-availability?username=${username}`,
+    {
+      credentials: 'include',
+      headers: {
+        ...FETCH_CONFIG.headers
+      },
```

Comment:
```suggestion
      headers: FETCH_CONFIG.headers,
```

This comment is just a suggestion, if you prefer stay as it is, you can resolve the conversation.